Mixmatch/libml/train.py

```diff
@@ -246,13 +246,15 @@ def eval_stats(self, batch=None, feed_extra=None, classify_op=None):
         self.train_print('kimg %-5d  accuracy train/valid/test  %.2f  %.2f  %.2f' %
                          tuple([self.tmp.step >> 10] + accuracies))
         
-        with open("../predictions.txt", "w") as text_file:
-            text_file.write(str(predicted) + "\n")
-#         self.train_print(str(labels))
-#         self.train_print('------------ predicted: ' + predicted)
+        # Saving predictions and labels to csv for further inspection
+        np.savetxt("prediction.csv", predicted, delimiter=",")
```

Comment:
It makes more sense to me to put them into a single csv, so that the prediction-label correspondence is more clear, e.g. for each entry it can be [label, pred0, pred1, ...]

Reply:
Done